src/client/App.js

```diff
@@ -11,13 +11,32 @@ import Header from './components/NavigationHeader/NavigationHeader';
 import Profile from './containers/Profile';
 import Channel from './containers/Channel/Channel';
 import Loader from './components/Loader/Loader';
+import fetchWithAuth from './utils/fetchWithAuth';
 import { UserContext } from './context/userContext';
 
 function App() {
+  const [currentUser, setCurrentUser] = useState();
   const { isAuthenticated, isLoading, user } = useAuthentication();
+
+  useEffect(() => {
+    const fetchUserFromDatabase = async () => {
+      try {
+        const { uid } = await user;
+        const url = `/api/users/current/?uid=`;
+        const userFromDB = await fetchWithAuth(url + uid);
+        return setCurrentUser(userFromDB);
+      } catch (err) {
+        return <p>{err}</p>;
```

Comment:
You cannot return this from `useEffect`. The only thing you can return is a cleanup function.
If you need to show an error, use another state variable called `err` with `useState`.